app/commands/incident.py

```diff
@@ -298,7 +298,11 @@ def submit(ack, view, say, body, client, logger):
     say(text=text, channel=channel_id)
 
     # Reminder to brief up
-    text = ":rotating_light: If this is a cybersecurity incident, please initiate the briefing process for CCCS and TBS OCIO Cyber"
+    text = ":one: Is this a `cybersecurity incident`? Please initiate the briefing process for CCCS and TBS OCIO Cyber"
```

Comment:
is it worth it to link this https://www.canada.ca/en/government/system/digital-government/online-security-privacy/security-identity-management/government-canada-cyber-security-event-management-plan.html#toc8 ?

Reply:
I heard the GCCSEMP got revamped as well

Reply:
Since it's not an exhaustive list and it's just a set of examples we might want to add more information since we could potentially under-report if they go by that list that is mainly for network security and not for application security.

Reply:
I like the idea of adding some examples - we could put them right in the message itself to give people a nudge. This is what comes to mind that we'd want to report on:
- secret leak
- targeted attack
- data or system compromise

Anything else?

Reply:
~data~ account or system compromise
data breach

We might be opening a can of worms in that tiny message block, but it's definitely worth explaining a bit about what we mean by a cyber incident. Also, do we want to raise an alarm for any security event? Because we could clarify that the examples listed are instances when we suspect potential leaks, breaches, etc.

Reply:
Heh possibly for the 🪱 - I've updated the message to look like so and we can adjust if it's not working:
![image](https://github.com/cds-snc/sre-bot/assets/2110107/46c57520-6d16-4e8c-90a0-4e1a45927411)

For raising an alarm, perhaps we could work with the Security team to have them help triage new incidents rather than leaving the onus on the product team to decide.